internal/cli/home_test.go

```diff
@@ -15,7 +15,11 @@ func TestHomeDir(t *testing.T) {
 
 	t.Run("Should return the home dir properly", func(t *testing.T) {
 		home, err := homeDir()
-		u.MustMatch(t, cmp.Diff(nil, err))
-		u.MustMatch(t, cmp.Diff(fmt.Sprintf("./%s", profileDir), home))
+		assert.Nil(t, err)
+		assert.Match(t, fmt.Sprintf("./%s", profileDir), home)
 	})
 }
+
+func TestNPM(t *testing.T) {
+	assert.True(t, strings.Contains("a long string", "long"), "expected 'long' to be a substring of 'a long string'")
```

Comment:
what's this testing?

Reply:
testing the reviewer?  :bushes:  started with this when fooling around with `go-cmp` options and stuff, I'll remove this.  Thanks for catching!